### PR TITLE
Revise TLS instructions

### PR DESCRIPTION
**Pull request by pierwill**
This revision reorganizes the instructions or signed and self-signed certificates into one instruction set.

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/security/enable-tls.md [hunk 1]
```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
...
+1. **Download or generate certificate files**
 
-1. **Install the certificate**
+    If using a certificate provided by a CA, follow their instructions to download the certificate files.
 
-    Place the private key file (`.key`) and the signed certificate file (`.crt`) in the `/etc/ssl/` directory.
+    If using a self-signed certificate, use the `openssl` utility to create a certificate.
+    The following command generates a private key file (`.key`) and a self-signed
+    certificate file (`.crt`) which remain valid for the specified `NUMBER_OF_DAYS`.
+    It outputs those files to `/etc/ssl/` and gives them the required permissions.
```
Comment (kelseiv) [inline content/v2.0/security/enable-tls.md]: If using a self-signed...., run the following to create a certificate:
```sh
.....
```
what info does command prompt for...valid number of days? directory to save files? 
`openssl` generates a private key file (`.key`) and a self-signed certificate file (`.crt`) with required permissions and saves them to `/etc/ssl/`. Files remain valid for the specified `NUMBER_OF_DAYS`.

File: content/v2.0/security/enable-tls.md [hunk 2]
```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
...
+    Unlike CA-signed certificates, self-signed certificates only provide cryptographic security to HTTPS requests.
+    They do not allow clients to verify the identity of the InfluxDB server.
+    With this certificate option, every InfluxDB instance requires a unique self-signed certificate.
+    You can generate a self-signed certificate on your own machine.
 
 <!-- InfluxDB supports certificates composed of a private key file (`.key`) and a signed certificate file (`.crt`) file pair, -->
 <!-- as well as certificates that combine the private key file and the signed certificate file into a single bundled file (`.pem`). -->
 
-## Enable HTTPS with a CA-signed certificate
+## Setup InfluxDB to use TLS over HTTPS
```
Comment (kelseiv) [inline content/v2.0/security/enable-tls.md]: Set up InfluxDB...

File: content/v2.0/security/enable-tls.md [hunk 3]
```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
 Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
 When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
 
+This pages outlines how to set up TLS over HTTPS with InfluxDB using either a signed or self-signed certificate.
```
Comment (kelseiv) [inline content/v2.0/security/enable-tls.md]: To set up TLS over HTTPS, do the following:
- Obtain [requirements](#requirements)
- [Set up InfluxDB....](#set-up...)

File: content/v2.0/security/enable-tls.md [hunk 4]
```diff
@@ -13,44 +13,61 @@ v2.0/tags: [security, authentication, tls, https, ssl]
 Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
 When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
```
Comment (kelseiv) [inline content/v2.0/security/enable-tls.md]: When the InfluxDB server is configured with a signed certificate, clients can verify the authenticity of the InfluxDB server?

File: content/v2.0/security/enable-tls.md (hunk 3 above)
Comment (pierwill) [reply content/v2.0/security/enable-tls.md]: I'm thinking of going with "Configure InfluxDB to use TLS"—

File: content/v2.0/security/enable-tls.md (hunk 4 above)
Comment (pierwill) [reply content/v2.0/security/enable-tls.md]: 👍

File: content/v2.0/security/enable-tls.md [hunk 5]
```diff
@@ -10,47 +10,67 @@ menu:
 v2.0/tags: [security, authentication, tls, https, ssl]
 ---
 
-Enabling HTTPS encrypts the communication between clients and the InfluxDB server.
-When configured with a signed certificate, HTTPS can also verify the authenticity of the InfluxDB server to connecting clients.
+Enabling TLS encrypts the communication between clients and the InfluxDB server.
+When configured with a signed certificate, TLS also allows clients to verify the authenticity of the InfluxDB server.
+
+To set up TLS over HTTPS, do the following:
```
Comment (kelseiv) [inline content/v2.0/security/enable-tls.md]: Looks good!

File: content/v2.0/security/enable-tls.md (hunk 1 above)
Comment (kelseiv) [reply content/v2.0/security/enable-tls.md]: @pierwill what do you think about this one?

Comment (pierwill) [reply content/v2.0/security/enable-tls.md]: Missed it on the first time around, looking at it now—

**Pull request merged**